static/assets/glyph-bookmark-unknown-16.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```

Comment:
On Thu, Apr 14, 2016 at 6:03 PM, Jordan Santell <notifications@github.com>
wrote:

> In static/assets/glyph-bookmark-unknown-16.svg
> <https://github.com/mozilla/tofino/pull/130#discussion_r59815436>:
>
> > @@ -0,0 +1,13 @@
> > +<?xml version="1.0"?>
>
> We already have a hollowed and filled bookmark star, what's this one for?
>
Sorry, future -- when we first start navigating, we don't know the final
location.  Just like in desktop, we should go to some "provisional" state,
and then mark hollow or solid.  That is, we'll go to a tristate (UNKNOWN,
BOOKMARKED, UNBOOKMARKED) and be more careful to reflect the WebView's
events in future.